tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -75,27 +76,34 @@ fn test_renew_lease<T: Simulator>(cluster: &mut Cluster<T>) {
     if cluster.engines.len() > 1 {
         // Wait for the leader lease to expire.
         thread::sleep(max_lease);
+        // Wait for the raft tick renew leader lease
+        thread::sleep(cluster.cfg.raft_store.raft_base_tick_interval.0);
```

Comment:
Why? Now that hibernate region is disabled, lease should be renewed infinitely, so there should be no expire time and no need to sleep another time.

Reply:
Maybe you need a extra test to show lease is extended infinitely. The original case should be kept for testing the behavior when infinite lease is disabled.